Cython/Includes/cpython/unicode.pxd

```diff
@@ -23,6 +23,21 @@ cdef extern from *:
     # New in version 3.3.
     Py_ssize_t PyUnicode_GET_LENGTH(object o)
 
+    Py_UCS1 *PyUnicode_1BYTE_DATA(object o)
+    Py_UCS2 *PyUnicode_2BYTE_DATA(object o)
+    Py_UCS4 *PyUnicode_4BYTE_DATA(object o)
+
+    int PyUnicode_WCHAR_KIND
```

Comment:
This is part of the 3.10 deprecation cycle and removed in 3.12. Even if we don't copy the whole documentation, I think that's still worth mentioning.